### ISSUE TITLE
eQTLS in a region view

### ISSUE THREAD

**Issue by grosscol** [statgen/bravo_vue#32]
Answers the questions:
- Which genes in the region have known eQTLs?
- Which variants in the region are eQTLs?
- What is the total number of eQTLs in the region?
- What is the total number of eQTLS in the region from each tissue type?

Feature is done when:
- Region page has a information card with summary counts of eQTLs in the region
- The gene bar component indicates which genes have known eQTLs

**Comment by grosscol** [comment]
Sketch of in-view tab navigation and tab contents

![Image](https://github.com/user-attachments/assets/22c09647-b4d2-41b0-99b6-1f540319380f)

**Comment by shawnkru** [comment]
For region view, we discussed having a visual region (searchable) but having an asterisks or any other symbol to indicate a possible QTL

**Comment by grosscol** [comment]
Good point.  The questions and is done when criteria should be updated to reflect that.  We could reuse the gene bar and bp bar components or derive a subclass of them to do something like the following:

![Image](https://github.com/user-attachments/assets/4382617a-af68-493d-a523-efeba1763af7)

**Comment by shawnkru** [comment]
So the idea is that the annotation marks are either not in the viewed region or too densely packed via pixels allowed when they are in the region view.  I'm just brainstorming, could we have a toggle a view limit, where it only flags ones within this window and have a note state x amount eQTLs are outside this viewing area.

<img width="468" alt="Image" src="https://github.com/user-attachments/assets/0a49ef49-d3da-4168-bf9f-6e1d44e3bef4" />

**Comment by shawnkru** [comment]
For the too densely packed eQTLs "stars" could we do a histogram style layover to show how many are in that exact region?

<img width="468" alt="Image" src="https://github.com/user-attachments/assets/6dca36ff-a643-4fcb-9b5b-4d09b7465222" />

**Comment by shawnkru** [comment]
For the number of stars in the same area, could we color code the different hits based on tissue or something?  Just running ideas

**Comment by shawnkru** [comment]
<img width="468" alt="Image" src="https://github.com/user-attachments/assets/580a5d52-69b4-469d-a846-ee96b4f85392" />

**Comment by grosscol** [comment]
@shawnkru One of the issues when the transcript an eqtl affects isn't in the region, is that there won't be a row for it in the visualization.

An issue for the eqtls that do have their affected transcript in range is that the position is often inside the transcript.  So we'd have to figure out how to indicate that without confusing the transcript bars.

I'm just putting in a histogram to answer the question "where are eQTLs in this region?" and letting the eqtl table and info cards in the eqtl tab handle the other questions.

**Comment by shawnkru** [comment]
Awesome, Im glad the histogram worked for displaying multiple hits in the same region.
